### ISSUE TITLE
electricityMeter can only use the default settings

### ISSUE THREAD

**Issue by script0803** [Koenkk/zigbee-herdsman-converters#8395]
extend: [identify(), onOff({"powerOnBehavior":false}), electricityMeter({
        threePhase: true,
        producedEnergy: true,
        acFrequency: true,
        powerFactor: true,
        })],

I used this code and applied it to my external converter, but none of these extra settings worked

**Comment by Koenkk** [comment]
Are you using the latest z2m?

**Comment by script0803** [comment]
Oh, I just updated my z2m and it works now, thank you very much.

In addition, I have another problem. I used
 
await endpoint.saveClusterAttributeKeyValue('haElectricalMeasurement', {
acPowerMultiplier: 1, acPowerDivisor: 1}); 

in the converter to change my power unit from kw to w, but it doesn't seem to work now.

**Comment by Koenkk** [comment]
To set the divisor with modern extend, see https://github.com/Koenkk/zigbee-herdsman-converters/blob/7c895668fde39734e0faa61f29bd324e7673837f/src/devices/robb.ts#L102

**Comment by script0803** [comment]
Hi， @Koenkk 

I tried these methods, but none of them worked. It seems that the divisor defined in my device will force overwrite what I set in the converter.  

So can I choose not to change the divisor but change the unit from w to kw?

<img width="330" alt="image" src="https://github.com/user-attachments/assets/254e25af-4823-478e-9a72-8951f03c3f54">

```javascript
const {electricityMeter} = require('zigbee-herdsman-converters/lib/modernExtend');
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const e = exposes.presets;

const definition = {
    zigbeeModel: ['SPM01X001'],
    model: 'SPM01X00111',
    vendor: 'BITUO TECHNIK',
    description: 'Automatically generated definition',
    configure: async (device, coordinatorEndpoint) => {
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['haElectricalMeasurement', 'seMetering']);
            await reporting.readMeteringMultiplierDivisor(endpoint);
            await endpoint.saveClusterAttributeKeyValue('haElectricalMeasurement', {
            acPowerMultiplier: 1, acPowerDivisor: 1,
        });
    },
    extend: [electricityMeter({fzMetering: fz.metering,
        acFrequency: true,
        powerFactor: true,
        configureReporting: false,
        producedEnergy: true,
        power: {divisor: 1, multiplier: 1},
        })],
    meta: {},
    exposes: [
        e.power_apparent()
    ],
};

module.exports = definition;
```

**Comment by script0803** [comment]
Could it be that after the devices are paired successfully, the converter changes the divisor internally, but after the converter 
is configured, the device actively reports all the information about the multiplier and divisor again, so the data changed by the converter is overwritten again?

![image](https://github.com/user-attachments/assets/02ec0826-3b0a-42e3-9215-b8cc306433ce)

**Comment by Koenkk** [comment]
>  the device actively reports all the information about the multiplier and divisor again, so the data changed by the converter is overwritten again?

Yes! You can confirm that by looking at your `data/database.db` file. But isn't this caused by ` await reporting.readMeteringMultiplierDivisor(endpoint);` ? This reads the divisor from the device.

**Comment by script0803** [comment]
`await reporting.readMeteringMultiplierDivisor(endpoint);` is used to read the coefficients in the meter, not the coefficients in ElectricalMeasurement. These two should be independent? 

And if I use `await reporting.readEletricalMeasurementMultiplierDivisors(endpoint);` to get the coefficients, then it should be read attribute in the data, not the current report attribute.

So I can only choose to write a converter to force EletricalMeasurementMultiplierDivisors to 1.
